node/services/config.go

```diff
@@ -22,6 +22,8 @@ type Config struct {
 	// Note: The trusted does *not* imply Headers are not verified, but trusted as reliable to fetch headers
 	// at any moment.
 	TrustedPeers []string
+	// BlockTime is the block time of the network.
+	BlockTime time.Duration
```

Comment:
I am still kind of iffy on this being a config value. While it doesn't really impact anything other than the frequency at which the syncer requests a new objective head from its trustedPeer, if we do end up using this config param elsewhere in the code, we would have to be very cautious on how it is used. Wdyt @Wondertan ?

Reply:
We should not make it part of the config value but a part of `params` pkg(#790). Taken from the params,  the value then gets pass to the Syncer

Reply:
Similarly, I don't think we should make it an option on the node, as by now.

Reply:
pls check: https://github.com/celestiaorg/celestia-node/pull/978/commits/9d897749c65fc3fb5e2e0fc8b80cecfd7bcb6bae